### PR TITLE
Include ability to log more items that were not in training set

### PR DESCRIPTION
**Pull request by iamshwin**
Log more (or fewer) items when warning about labels not existing in the training set, but existing in the validation set.

### CODE REVIEW

**Comment by sgugger** [comment]
I don't see why not, thanks for your PR.
